cheroot/connections.py

```diff
@@ -133,10 +137,9 @@ def get_conn(self, server_socket):
         ss_fileno = server_socket.fileno()
         socket_dict[ss_fileno] = server_socket
         try:
-            rlist, _, _ = select.select(list(socket_dict), [], [], 0.1)
-            # No available socket.
-            if not rlist:
-                return None
+            for sock in socket_dict:
+                self._selector.register(sock, selectors.EVENT_READ)
+            key_events = [key.fd for key, _event self._selector.select(timeout=0.1)]
```

Comment:
:laughing: "coding" in browser is hard:
```suggestion
            key_events = [key.fd for key, _event in self._selector.select(timeout=0.1)]
```
(LGTM has found this mistake @ https://lgtm.com/projects/g/cherrypy/cheroot/rev/pr-3818976e81a56bcb93f77451590a1d1acc801594 / https://github.com/cherrypy/cheroot/pull/301#issuecomment-660675296)